docs/enroute/STH.md

```diff
@@ -116,6 +116,14 @@ If a sequencing conflict is to occur, STH shall coordinate with CH TMA as to an
 
 STH may clear aircraft direct to the STAR's CH TMA boundary without coordination, provided that they have been cleared to rejoin the STAR thereafter. Aircraft may be cleared to track direct to a fix within CH TMA's boundary with coordination, subject to the same condition.
 
+#### Short STARs - NZCH
+
+`BELEE #J` and `BELEE #K` are Short STARs into CH TMA that capture ILS 02 and ILS 20 at their intermediate fixes, `IDUBU` and `DIVSU` respectively. Controllers must co-ordinate with CH TMA or CH TWR before clearing aircraft for these arrivals, as prior planning may be required to ensure flow is maintained.
```

Comment:
Coordination would only need to occur with TMA as the primarily affected sector. TMAs can happily have aircraft intercept the midpoint without coord from the ADC.